content/c-sharp/concepts/strings/terms/split/split.md

```diff
@@ -0,0 +1,136 @@
+---
+Title: '.Split()'
+Description: 'Divides a string into an array of substrings based on specified delimeter(s).'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Libraries'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Split()`** method breaks a string into substrings based on specified delimiter(s), which can be a character, an array of characters, or a string. It returns an array of strings containing the split substrings.
+
+## Syntax
+
+The various overloads of the `.Split()` method in C# are as follows:
```

Comment:
You mention "overloads" of the .Split() method, but it might help to explicitly state that these are different ways the method can be called, depending on the type of input parameters.

```suggestion
The `.Split()` method can be used in the following ways:
```